### DISCUSSION TITLE
Missing guides from the old wiki

### DISCUSSION THREAD

**Discussion by ilker-aktuna** [Guilouz/Creality-Helper-Script-Wiki#432] [Issues]
The old wiki had these items:
Input XY shaping
https://github.com/Guilouz/Creality-K1-and-K1-Max/wiki/Fix-issue-with-Input-Shaper?fbclid=IwAR23dsCO0Xz4UiiGH2OQlb0ARb7YjY5xMCxIm1fmWHQmGfk5dM5kZiMxKqY 

Pid Tuning & other macros
https://github.com/Guilouz/Creality-K1-and-K1-Max/wiki/Useful-Macros?fbclid=IwAR2sxjVn03l9s0eJdgIaB5rvjlwBlDz4mukTBA1XLm_jzUgpPFLIjfR4Ud8 

Improve Bed level & manual bed leveling
https://github.com/Guilouz/Creality-K1-and-K1-Max/wiki/Improve-Bed-Mesh

I can't find these in the new wiki. Where can I find these ?

**Comment by Guilouz** [comment]
https://guilouz.github.io/Creality-K1-Series/helper-script/improved-shapers-calibrations/

https://guilouz.github.io/Creality-K1-Series/helper-script/useful-macros/

**Comment by ilker-aktuna** [comment]
thanks.
